algolia/search/rendering_content_test.go

```diff
@@ -29,10 +32,92 @@ func TestUnmarshalRenderingContent(t *testing.T) {
 	err := json.Unmarshal([]byte(payload), &r)
 	require.NoError(t, err)
 	require.Equal(t, r.FacetOrdering.Facets.Order, []string{"brand", "size", "color"})
+
 	require.Equal(t, r.FacetOrdering.Values["brand"].Order, []string{"Apple", "Sony", "Samsung"})
 	require.Equal(t, *r.FacetOrdering.Values["brand"].SortRemainingBy, Alpha)
+
 	require.Equal(t, r.FacetOrdering.Values["color"].Order, []string{"red", "green"})
 	require.Nil(t, r.FacetOrdering.Values["color"].SortRemainingBy)
+
+	require.Nil(t, r.FacetOrdering.Values["size"].Order)
+}
+
+func TestMarshalRenderingContent(t *testing.T) {
+
+	tests := []struct {
+		name     string
+		input    RenderingContent
+		expected string
+	}{
+		{
+			name: "empty values",
+			input: RenderingContent{
+				FacetOrdering: &FacetOrdering{
+					Facets: &FacetsOrder{
+						Order: []string{"brand", "size", "color"},
+					},
+					Values: nil,
+				},
+			},
+			expected: `{
+	"facetOrdering": {
+		"facets": {
+			"order": [
+				"brand",
+				"size",
+				"color"
+			]
+		}
+	}
+}`,
```

Comment:
You can write the JSON with the expected indentation and use `require.JSONEq()` for the assertion.